precompile-script.py

```diff
@@ -127,12 +128,12 @@ def convert_stream(fin, fout):
                 stage_name = section_name[:-6]
                 converter = Converter('spirv',
                                       STAGE_MAP[stage_name],
-                                      fout)
+                                      fout, spirv_as)
```

Comment:
I have a habit of putting each unrelated argument on a separate line whenever the line is too long. It might be nice to stick to that and put `spirv_as` on its own line, otherwise it looks odd.